Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,43 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox
+    {
+        public Size Size { get; set; } //default set to 0 in the enum cant be null
+
+        public (int, int) Location { get; set; } //cant be null because value
+
+        //and a Person Owner
+        public Person Owner; //cant be null because struct
+
+        public Mailbox(Size Size, (int, int) Location, Person Owner)
```

Comment:
```suggestion
        public Mailbox(Size size, (int, int) location, Person owner)
```
Parameters should be camel case

Reply:
I have two different branches going on for this assignment. I made your corrections in one but not the other. waiting to merge them correctly. Thank you for your input!